README.md

```diff
@@ -207,3 +207,7 @@ MIT License - see [LICENSE](LICENSE) file for details.
 This action automatically logs out of Tailscale when the job completes, ensuring no persistent connections remain. For OAuth authentication, connections are ephemeral by default.
 
 For security issues, please see our [security policy](SECURITY.md).
+
+## CI Notes
+
+CI tests run against the tailscalegithubactionbot.github tailnet. Check our usual credential store for credentials.
```

Comment:
Drive-by, note to remind maintainers about which tailnet is being used for tests and where to find credentials.